server/aap/commands/__init__.py

```diff
@@ -15,4 +15,32 @@
 from .fix_items_expired import FixItemsExpiry  # noqa
 from .export_legal_archive_to_archived import ExportLegalArchiveToArchived  # noqa
 from .export_to_newroom import ExportToNewsroom  # noqa
-from .import_sport_calendar import ImportSportCalendarDoc  # noqa
\ No newline at end of file
+from .import_sport_calendar import ImportSportCalendarDoc  # noqa
+from .fulfill_image_assignments import FullfillImageAssignments  # noqa
+from aap.commands import FullfillImageAssignments
```

Comment:
There are two imports for ` FullfillImageAssignments` here, only 1 is required

Reply:
OK